sentences/fr/weather_HassGetWeather.yaml

```diff
@@ -3,12 +3,53 @@ intents:
   HassGetWeather:
     data:
       - sentences:
-          - "(<quelest>|donne[s][-moi]) (le|la|les) (temps|météo)"
-          - "(Donne[s]( |-)moi|Dis( |-)moi) le temps qu'il fait"
+          - "(<quelest>|<donnemoi>) (le|la|les) (temps|météo)"
+          - "<donnemoi> le temps qu'il fait"
           - "Quel temps fait-il"
       - sentences:
-          - "(<quelest>|donne[s][-moi]) (le|la|les) (temps|météo) (pour|à) [<le>]{name}"
-          - "(Donne-moi|Dis-moi) le temps qu'il fait à [<le>]{name}"
+          - "(<quelest>|<donnemoi>) (le|la|les) (temps|météo) (pour|à) [<le>]{name}"
+          - "<donnemoi> le temps qu'il fait à [<le>]{name}"
           - "Quel temps fait-il à [<le>]{name}"
         requires_context:
           domain: weather
+
+      - sentences:
+          - "<donnemoi> (un[e]|le) [bulletin|rapport] météo [détaillé[e]]"
```

Comment:
Just realised with so many optional words, it collides with "donne-moi la météo". Will split in two sentences.